### PR TITLE
Back/feat/19 api server routes

### PR DESCRIPTION
**Pull request by rgladue**

### CODE REVIEW

**Comment by rgladue** [comment]
Looks like the package.json error thingy is happening again with this branch

File: server/routes/students.js [hunk 1]
```diff
@@ -9,5 +9,19 @@ module.exports = (db) => {
...
   });
 
+  //return JSON with student object OR null if not found
+  router.post('/api/student/login', (req, res) => {
+    console.log(req.body);
+    const accessCode = req.body.access_code;
+    const query = "SELECT * FROM students WHERE access_code = $1", [accessCode];
+
+    db.query(query)
+      .then(data => {
```
Comment (kurtisgrant) [inline server/routes/students.js:19]: You want to include your query parameters as the second argument in the db.query method. 
Lines 16-18 should read:
```
    const query = "SELECT * FROM students WHERE access_code = $1";

    db.query(query, [accessCode])
      .then(data => {
```

File: server/routes/teachers.js [hunk 2]
```diff
@@ -9,5 +9,17 @@ module.exports = (db) => {
       .catch(e => console.log(e.message))
   });
 
+  //return JSON of teacher object OR null if not found
+  router.post('/api/teacher/login', (req, res) => {
+    const email = req.body.email;
+    const query = "SELECT * FROM teachers WHERE email = $1", [email];
+    db.query(query)
+      .then(data => {
```
Comment (kurtisgrant) [inline server/routes/teachers.js:17]: Same thing here, `[email]` should be moved into `db.query(query, [email])` :)

**Review by kurtisgrant — CHANGES_REQUESTED** [review]
Good start :) See the comments I added about including query parameters in the `.query` method

**Pull request closed without merging**